pr_agent/algo/ai_handler.py

```diff
@@ -68,22 +71,29 @@ def deployment_id(self):
         """
         return get_settings().get("OPENAI.DEPLOYMENT_ID", None)
 
-    @retry(exceptions=(APIError, Timeout, TryAgain, AttributeError, RateLimitError),
-           tries=OPENAI_RETRIES, delay=2, backoff=2, jitter=(1, 3))
-    async def chat_completion(self, model: str, system: str, user: str, temperature: float = 0.2):
+    @retry(
+        exceptions=(APIError, TryAgain, AttributeError, RateLimitError),
+        tries=OPENAI_RETRIES,
+        delay=2,
+        backoff=2,
+        jitter=(1, 3),
+    )
```

Comment:
**Suggestion:** It's a good practice to avoid hardcoding values directly in the code. The values `2` and `3` in the `jitter` parameter of the `retry` decorator are hardcoded. Consider moving these values to a configuration file or defining them as constants at the top of your file. This will make it easier to manage and modify these values in the future. [best practice]
```suggestion
    JITTER_MIN = 1
    JITTER_MAX = 3
    @retry(
        exceptions=(APIError, TryAgain, AttributeError, RateLimitError),
        tries=OPENAI_RETRIES,
        delay=2,
        backoff=2,
        jitter=(JITTER_MIN, JITTER_MAX),
    )
```